cartography/data/jobs/cleanup/aws_account_dns_cleanup.json

```diff
@@ -4,11 +4,6 @@
     "query": "MATCH (n:AWSDNSRecord)-[:MEMBER_OF_DNS_ZONE]->(:AWSDNSZone)<-[:RESOURCE]-(:AWSAccount{id: {AWS_ID}}) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
     "iterative": true,
     "iterationsize": 100
```

Comment:
This cleanup job runs at the [end of a single AWS account's sync](https://github.com/lyft/cartography/blob/c39a3bbfc1e8a7f8a6a04658467508b644db1c89/cartography/intel/aws/__init__.py#L48).  This statement is no longer necessary because the nodes are now explicitly deleted in aws_dns_cleanup.json below.